src/libraries/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -564,4 +567,7 @@
   <data name="net_http_chunk_too_large" xml:space="preserve">
     <value>The HTTP/1.1 response chunk was too large.</value>
   </data>
-</root>
\ No newline at end of file
+    <data name="net_unsupported_extended_connect" xml:space="preserve">
+    <value>Failed to establish web socket connection over HTTP/2 because extended CONNECT is not supported. Try to downgrade the request version to HTTP/1.1</value>
```

Comment:
```suggestion
    <value>Failed to establish web socket connection over HTTP/2 because extended CONNECT is not supported. Try to downgrade the request version to HTTP/1.1.</value>
```
```suggestion
    <value>Failed to establish web socket connection over HTTP/2 because extended CONNECT is not supported. Try to downgrade the request version to HTTP/1.1</value>
```